src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/alert-list/alert-list.component.ts

```diff
@@ -1,20 +1,29 @@
 import { Component, OnInit, TemplateRef, ViewChild } from '@angular/core';
 import { I18n } from '@ngx-translate/i18n-polyfill';
 import { CellTemplate } from '../../../../shared/enum/cell-template.enum';
+import { CdTableAction } from '../../../../shared/models/cd-table-action';
 import { CdTableColumn } from '../../../../shared/models/cd-table-column';
 import { CdTableSelection } from '../../../../shared/models/cd-table-selection';
+import { Permission } from '../../../../shared/models/permissions';
 import { CdDatePipe } from '../../../../shared/pipes/cd-date.pipe';
+import { AuthStorageService } from '../../../../shared/services/auth-storage.service';
 import { PrometheusAlertService } from '../../../../shared/services/prometheus-alert.service';
+import { URLBuilderService } from '../../../../shared/services/url-builder.service';
+
+const BASE_URL = 'silence'; // as only silence actions can be used
 
 @Component({
   selector: 'cd-prometheus-list',
-  templateUrl: './prometheus-list.component.html',
-  styleUrls: ['./prometheus-list.component.scss']
+  providers: [{ provide: URLBuilderService, useValue: new URLBuilderService(BASE_URL) }],
+  templateUrl: './alert-list.component.html',
+  styleUrls: ['./alert-list.component.scss']
 })
-export class PrometheusListComponent implements OnInit {
+export class AlertListComponent implements OnInit {
   @ViewChild('externalLinkTpl')
   externalLinkTpl: TemplateRef<any>;
   columns: CdTableColumn[];
+  tableActions: CdTableAction[];
+  permission: Permission;
   selection = new CdTableSelection();
   customCss = {
     'label label-danger': 'active',
```

Comment:
Could we use green for the active state? we use green in other pages for positive states.
```suggestion
    'label label-success': 'active',
```

Reply:
Here the active state is even more critical than the active silence state. As we can assume the alert hasn't been seen by the admin yet or he didn't had the time to address the alert yet.